### PR TITLE
OH GAWD WHAT AM I DOING IM GONNA CRASH THE SYSTEM HALP ME I'VE NEVER DONE THIS BEFORE.

### PR DESCRIPTION
**Pull request by chococookies131**
FIXED SOME C ERROR MESSAGES

I LIKE CAPS LOCK.

### CODE REVIEW

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h [hunk 1]
```diff
@@ -14,10 +14,14 @@ void Nrt_setErrMsgIsAlreadyDeletedOrRemovedInternal();
...
+void Nrt_setErrMsgIsCharacterNotFoundInternal();
 void Nrt_setErrMsgCustomInternal(const char* message);
 
 #ifdef __cplusplus
 }
 #endif
 
-#endif //!NOVELRT_INTEROP_NOVELRTINTEROPERRORHANDLINGINTERNAL_H
\ No newline at end of file
+#endif //!NOVELRT_INTEROP_NOVELRTINTEROPERRORHANDLINGINTERNAL_H
```
Comment (RubyNova) [inline include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:27]: What actually changed here?

**Comment by jeuxjeux20** [comment]
Are you ok?

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h [hunk 2]
```diff
@@ -14,10 +14,14 @@ void Nrt_setErrMsgIsAlreadyDeletedOrRemovedInternal();
 void Nrt_setErrMsgIsNotSupportedInternal();
 void Nrt_setErrMsgIsInitialisationFailureInternal();
 void Nrt_setErrMsgIsFunctionNotFoundInternal();
+void Nrt_setErrMsgIsNotInitializedInternal();
```
Comment (RubyNova) [inline include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:17]: Please note, this is a UK english codebase. Initialized needs to be spelt as Initialised.

File: src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp [hunk 3]
```diff
@@ -8,8 +8,12 @@ const char* const errMsgIsNaN = "Unable to continue! Calculation resulted in an
 const char* const errMsgIsDivideByZero = "Unable to continue! Divide by zero was attempted.";
 const char* const errMsgIsNotSupportedInternal = "The current operation is not supported.";
 const char* const errMsgIsAlreadyDeletedOrRemoved = "Unable to continue! The specific item has already been deleted or removed.";
-const char* const errMsgIsInitialisationFailure = "Service could not be initialised.";
-const char* const errMsgIsFunctionNotFound = "Failed to locate the specified managed function.";
+const char* const errMsgIsInitialisationFailure = "Unable to continue! Service could not be initialised.";
+const char* const errMsgIsFunctionNotFound = "Unable to continue! Failed to locate the specified managed function.";
+const char* const errMsgIsNotInitialized = "Unable to continue! The service is currently not initialised.";
```
Comment (RubyNova) [inline src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp:13]: US spelling of Initialised here also needs to be changed.

**Comment by ismaelbrowne** [comment]
I THINK SO

**Review by RubyNova — CHANGES_REQUESTED** [review]
Only two minor changes and a question. Once these are addressed (and CI passes) I'll merge. :)

**Comment by ismaelbrowne** [comment]
Oop left caps lock on

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h (hunk 1 above)
Comment (ismaelbrowne) [reply include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:27]: Didn't change anything intentionally

File: include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h (hunk 2 above)
Comment (ismaelbrowne) [reply include/NovelRT.Interop/NrtInteropErrorHandlingInternal.h:17]: Ok

File: src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp (hunk 3 above)
Comment (ismaelbrowne) [reply src/NovelRT.Interop/NrtInteropErrorHandlingInternal.cpp:13]: KK

**Comment by RubyNova** [comment]
This is related to #247 . Looks like this addresses the custom exception messages. The ticket may need revisiting for stdlib errors.

**Review by RubyNova — APPROVED** [review]
LGTM, just waiting for CI to pass before merge.

**Pull request merged**